src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,313 @@
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get AppId from Info, Error - " + (StcError)ret);
+                    return string.Empty;
+                }
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// A property to get interface name from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>Interface name.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string InterfaceName
+        {
+            get
+            {
+                string ifaceName;
+                int ret = Interop.Stc.Info.GetInterfaceName(_infoHandle, out ifaceName);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get interface name from info, Error - " + (StcError)ret);
+                    return string.Empty;
+                }
+                return ifaceName;
+            }
+        }
+
+        /// <summary>
+        /// A property to get "from" value(start) of time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>from(start) of time interval.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public DateTime From
+        {
+            get
+            {
+                DateTime from;
+                DateTime to;
+                int ret = Interop.Stc.Info.GetTimeInterval(_infoHandle, out from, out to);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get time interval(from value) from info, Error - " + (StcError)ret);
+                }
+                return from;
+            }
+        }
+
+        /// <summary>
+        /// A property to get "to" value(end) of time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>to(end) of time interval.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
```

Comment:
Wrong comment. this property only provides getter. So when try set any value to this property. It makes a complication error.